preprocessors/openstreetmap.schema.json

```diff
@@ -0,0 +1,138 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/openstreetmap.schema.json",
+    "type": "object",
+    "title": "OpenStreetMap Preprocessor",
+    "description": "Information on the contents of a place on a map",
+    "properties": {
+
+        "timestamp": {
+            "type": "number",
+            "minimum": 0
+        },
+        "request_uuid": {
+            "type": "string",
+            "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-4[0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+        },
+        "name": {
+            "type": "string",
+            "pattern": "^[A-Za-z]{2,6}(\\.((?!-)[A-Za-z0-9-]+(?<!-)))+$"
+        },
+        "data": {
```

Comment:
`data`, `timestamp`, `request_uuid`, and `name` are not supposed to be here. This schema only describes what would be under the `data` key in the response, not the entire response itself. Please refer to other preprocessor schemas for examples.